documentation/getting-started/README.md

```diff
@@ -22,85 +22,79 @@ You can install TestCafe Studio on MacOS, Windows or Linux. See [Installation](.
 
 ## Record Test Actions
 
-First, create a directory to store your tests.
-Go to TestCafe Studio, open the **File** menu, click **Open Tests Directory** and navigate to the created directory.
-You can view the directory's contents in the [Explorer](../user-interface/explorer-panel.md) panel.
+1. First, create a [test directory](../guides/organize-tests.md#test-directory) to store your tests. Then go to TestCafe Studio, open the **File** menu, click **Open Tests Directory** and navigate to the created directory. You can view the directory's contents in the [Explorer](../user-interface/explorer-panel.md) panel.
 
-![The Explorer panel](../../images/getting-started/explorer-panel.png)
+    ![The Explorer panel](../../images/getting-started/explorer-panel.png)
 
-Next, create a test fixture for the sample webpage tests.
-In the **Explorer** panel, click the **Create a New Fixture** button, and in the invoked **New Fixture** dialog, specify the fixture name and the tested page's URL. Click **Create**.
+2. Create a [fixture](../guides/record-tests/README.md#create-fixtures) for the sample webpage tests. To do this, click the ![Create a New Fixture button](../../images/getting-started/create-new-fixture-icon.svg) **Create a New Fixture** button in the **Explorer** panel. This invokes the **New Fixture** dialog where you can specify the fixture name and the tested page's URL ([http://devexpress.github.io/testcafe/example](http://devexpress.github.io/testcafe/example)).
 
-![The New Fixture dialog](../../images/getting-started/new-fixture-dialog.png)
+    ![The New Fixture dialog](../../images/getting-started/new-fixture-dialog.png)
 
-The new fixture is added to the test directory.
+    Click **Create**. The new fixture is added to the test directory.
 
-To start recording a test, click the **Record a New Test** button in the **Explorer** panel.
-TestCafe Studio launches the default browser with the tested site and starts recording your activity on the webpage.
+3. Start recording a test. To do this, click the ![Record button](../../images/getting-started/record-test-icon.png) **Record a New Test** button in the **Explorer** panel. TestCafe Studio launches the default browser with the tested site and starts recording your activity on the webpage.
```

Comment:
We still cannot detect the default browser (look at https://github.com/DevExpress/testcafe-studio/issues/833)

Reply:
Ok, I'll add a new step where we describe how to select a record configuration